src/lambdapi.ml

```diff
@@ -111,6 +109,21 @@ let lsp_server_cmd : config -> bool -> string -> unit =
   let run _ = init cfg; Lsp.Lp_lsp.main standard_lsp lsp_log_file in
   Console.handle_exceptions run
 
+(** Printing a decision tree. *)
+let decision_tree_cmd : config -> (Path.t * string) -> unit =
+  fun cfg (mp, sym) ->
+  let run _ =
+    Timed.(verbose := 0); (* To avoid printing the "Checked ..." line *)
+    init cfg;
+    let sign = Compile.compile false mp in
+    let (sym, _) =
+      try StrMap.find sym Timed.(!(sign.sign_symbols))
+      with Not_found -> fatal_no_pos "Symbol not found."
```

Comment:
Last thing, here I would prefer:
```ocaml
    let sym =
      let sign = Compile.compile false mp in
      try fst (StrMap.find sym Timed.(!(sign.sign_symbols)))
      with Not_found -> fatal_no_pos "Symbol not found."
    in
```
This way the logic of the function is easier to identify (even though in that case it is pretty easy already, but I want to encourage that style).